### PR TITLE
Configure storybook to use Webpack 5

### PR DESCRIPTION
**Pull request by spiess-demos**
To mitigate issues arising from Webpack 4 (Storybook builder default) being installed alongside Webpack 5 (being the build tool for demosplan-ui), Storybook is configured to use Webpack 5, too.

Along the way, Storybook is upgraded to its latest version.

See:
- https://github.com/vuejs/vue-cli/issues/5986
- https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#fixing-hoisting-issues

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,20 +6,38 @@
   "description": "Vue components, Vue directives, Design Token and Scss files to build interfaces for demosPlan.",
   "main": "./dist/demosplan-ui.umd.js",
   "dependencies": {
+    "@babel/core": "^7.20.2",
```
Comment (salisdemos) [inline package.json:9]: there is a devDependency-Section again. f You update the branch with main, you will see it.

File: package.json [hunk 2]
```diff
@@ -28,40 +46,22 @@
...
-    "postcss": "^8.4.19",
-    "postcss-loader": "^7.0.1",
-    "style-loader": "^3.3.1",
-    "vue-loader": "^15",
-    "vue-template-compiler": "^2.7.14",
     "webpack": "^5.75.0",
     "webpack-cli": "^5.0.0"
   },
   "peerDependencies": {
+    "react": "16.14.0",
```
Comment (salisdemos) [inline package.json:60]: Don't have to be done right now I You are uncertain.
may react be a devDependency for Storybook. Or may we drop it? I know. I added it here....

File: package.json [hunk 3]
```diff
@@ -73,5 +73,9 @@
     "build:dev": "webpack --mode=development",
     "build:prod": "webpack --mode=production --define-process-env-node-env=production",
     "watch": "webpack --watch"
+  },
+  "devDependencies": {
```
Comment (salisdemos) [inline package.json:77]: see above. Or: Why are all the babels et al not in here?

**Review by salisdemos — CHANGES_REQUESTED** [review]
please update with main

**Comment by spiess-demos** [comment]
Closed in favor of https://github.com/demos-europe/demosplan-ui/pull/36 because the rebase made it difficult to merge.

**Pull request closed without merging**